packages/devtools_app/lib/src/shared/diagnostics/object_group_api.dart

```diff
@@ -0,0 +1,43 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'package:vm_service/vm_service.dart';
+
+import '../eval_on_dart_library.dart';
+import 'primitives/instance_ref.dart';
+
+abstract class ObjectGroupApi<T extends DiagnosticableTree>
```

Comment:
Should this class really be `InspectorObjectGroupApi`? It seems all these abstract methods have to do with the inspector and aren't really general